api/app/serializers/spree/v2/storefront/product_serializer.rb

```diff
@@ -59,11 +59,16 @@ class ProductSerializer < BaseSerializer
           record_type: :image,
           serializer: :image
 
-        has_one  :default_variant,
+        has_one :default_variant,
           object_method_name: :default_variant,
           id_method_name: :default_variant_id,
           record_type: :variant,
           serializer: :variant
+
+        has_one :master,
```

Comment:
@szymoniwacz after 2nd thaught let's rename this to `master_variant`

Reply:
@szymoniwacz line 9 - please add `sku` attribute there

Reply:
I'm not happy with that `master` naming, which can be offensive in some cultures. 

https://www.theserverside.com/opinion/Master-slave-terminology-alternatives-you-can-use-right-now

`primary` is a good candidate to replace it, let's start with this serializer

Reply:
@damianlegawiec 
I've added `:sku` in this PR: https://github.com/spree/spree/pull/11266/files.
You're right about _master_ naming.